packages/react-components/react-storybook-addon/src/theme.ts

```diff
@@ -8,8 +8,6 @@ import {
 
 import type { Theme } from '@fluentui/react-theme';
 
-export { FluentProvider } from '@fluentui/react-provider';
```

Comment:
re-exports from already aliased module causes issues to node module `register` overrides.

tested with:
- `ts-node/register`
- `@babel/register`